notebooks/Accessing_File_with_GDS.ipynb

```diff
@@ -253,11 +248,13 @@
     "read_count = fd.pread(torch_arr.data_ptr(), 10, 3)      # read 10 bytes starting from file offset 3\n",
     "print(\"{:10} cnt: {}  content: {}\".format(\"torch_arr\", read_count, torch_arr))\n",
     "fd.close()\n",
+    "os.close(fno)\n",
     "\n",
     "fno = os.open(\"output.raw\", os.O_RDWR | os.O_CREAT | os.O_TRUNC)\n",
     "fd = CuFileDriver(fno)\n",
     "write_count = fd.pwrite(np_arr, 10, 5)      # write 10 bytes from np_array to file starting from offset 5\n",
     "fd.close()\n",
+    "os.close(fno)\n",
```

Comment:
Might be worth writing this to use [a context manager (`with` statement)]( https://docs.python.org/3/tutorial/inputoutput.html#reading-and-writing-files ). Just a thought 🙂

Reply:
Thanks for the review!

This is to showcase the different ways to call the methods.
Here, I couldn't use with statement for `os.close(fno)` because Python's built-in os.open() returns `int` type and there is no context manager available.

Reply:
We can always [roll our own]( https://docs.python.org/3/library/contextlib.html#contextlib.contextmanager ). Though maybe that's unnecessary for an example.

Reply:
I am not sure if we want to provide such a wrapper in Python for the file descriptor(int).

`with` is supported by our API. the recommended way for using it (without close method) is described in the notebook and a description of the GDS reader page so don't need to change here?

https://github.com/NVIDIA/MagnumIO/tree/main/gds/readers/cucim-gds

```python
with fs.open("output.raw", "w") as fd:
    # Write 10 bytes from cp_array to file starting from offset 5
    write_count = fd.pwrite(cp_arr, 10, 5)
```